language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/helpers/TransactionsServiceImpl.scala

```diff
@@ -55,6 +55,7 @@ final class TransactionsServiceImpl(ledgerContent: Observable[LedgerItem])
         Metadata.Key.of("cause", Metadata.ASCII_STRING_MARSHALLER),
         s"BEGIN should be strictly smaller than END. Found BEGIN '${request.getBegin}' and END '${request.getEnd}'",
       )
+      // TODO error codes: Adapt V2 ?
```

Comment:
Language team?

Reply:
This is actually for the runtime team

Reply:
This one is for you @stefanobaghino-da

Reply:
Can you please create a ticket to track the required downstream changes that you are not in a position to address? Thanks.

Reply:
https://github.com/digital-asset/daml/issues/11587

Reply:
For the record, this is just a mock, so no need to adapt this. This is not (and definitely _should_ not) be used to test responses to errors, so no need to have the mock implement self-service error codes. The important bits have already been handled as part of https://github.com/digital-asset/daml/pull/11223